tests/Unit/API/Google/AdsCampaignTest.php

```diff
@@ -41,12 +43,15 @@ public function setUp() {
 
 		$this->ads_client_setup();
 
-		$this->ad_group    = $this->createMock( AdsGroup::class );
-		$this->budget      = $this->createMock( AdsCampaignBudget::class );
-		$this->options     = $this->createMock( OptionsInterface::class );
+		$this->ad_group      = $this->createMock( AdsGroup::class );
+		$this->budget        = $this->createMock( AdsCampaignBudget::class );
+		$this->options       = $this->createMock( OptionsInterface::class );
+		$this->wc            = $this->createMock( WC::class );
+		$this->google_helper = new GoogleHelper( $this->wc );
```

Comment:
I would add the `google_helper` as an `@property` in the class doc block. We don't really use `wc` as a property so I'd personally just pass it directly in the constructor.

```suggestion
		$this->google_helper = new GoogleHelper( $this->createMock( WC::class ) );
```

Reply:
Thanks! This is also updated in 0247998.